src/AppKit/Enums.cs

```diff
@@ -1223,7 +1223,11 @@ public enum NSUserInterfaceLayoutOrientation : nint {
 
 	// NSStackView.h:typedef float NSStackViewVisibilityPriority
 	public enum NSStackViewVisibilityPriority : int {
+#if !XAMCORE_4_0
 		Musthold = 1000,
+#else
+		MustHold = 1000,
+#endif
```

Comment:
To avoid future migration pains use this pattern instead:
```
MustHold = 1000,
#if !XAMCORE_4_0
[Obsolete ("Use 'MustHold' instead.")]
Musthold = MustHold,
#endif
```
This way there will be a warning (well in advance) and developers can fix them (long before) it becomes required by the new profile.